### ISSUE TITLE
Add support for command line prefix

### ISSUE THREAD

**Issue by Captn138** [killpowa/Twickt-Launcher#1551]
- [ x] I searched the issue tracker for other issues covering my case.

**Is your feature request related to a problem? Please describe.**
_I would like to be able to launch an instance with a particular command line prefix, for instance "prime-run" on Linux._

**Describe the solution you'd like**
_A way to configure an instance with a field where I can write my command line prefix._

**Describe alternatives you've considered**
_I tried launching GDlauncher with my arguments on command line but it doesn't work the way I expected._

**Additional context**
_Here's what it looks like in Lutris for example._
![image](https://user-images.githubusercontent.com/25801033/222842411-cbddfbe2-77d1-4a35-b7d1-02bcee840651.png)

**Comment by Eskaan** [comment]
Yeah, arguments made on GDL won't affect the Java instance as it gets spawned as separate process and disowned. I also thought about something like that before, but I would integrate it into the current java args as a full launch command with substitutions replaced by the launcher (like the java path and game args).
